src/Compilers/Core/MSBuildTaskTests/DotNetSdkTests.cs

```diff
@@ -0,0 +1,288 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.IO;
+using Roslyn.Test.Utilities;
+
+namespace Microsoft.CodeAnalysis.BuildTasks.UnitTests
+{
+    public class DotNetSdkTests : DotNetSdkTestBase
```

Comment:
>DotNetSdkTests [](start = 17, length = 14)

If the intent is to test MSBuild you need to be deploying a Directory.Build.props, Directory.Build.targets, and editorconfig that disables parent lookup. That's necessary to make these type of tests stable.

Reply:
Why editor config?

Reply:
Good point. I'll add those.